sysml/src/examples/State Space Representation Examples/CartSample.sysml

```diff
@@ -0,0 +1,61 @@
+// State Space Representation cart example
+
+package CartSample {
+    import StateSpaceRepresentation::*;
+    part def Cart {
+        attribute mass :> ISQ::mass;
+
+        attribute def CartInput :> Input {
+            attribute force :> ISQ::force;
+        }
+
+        attribute def CartOutput :> Output {
+            attribute velocity :> ISQ::speed;
```

Comment:
Minor comment is that according to ISO 80000-3 velocity is used for the vector quantity and speed for the scalar quantity, i.e. the name for the magnitude of the velocity, independent of direction. You may want to use `speed` in these examples.

Reply:
Thank you.  I did not know such difference.  This is 1D example and as you said, `speed` is relevant.  I used velocity in EV example but it subsets `ISQ::speed` as well.  Maybe we should have named it to speed.